app/views/errors.py

```diff
@@ -23,13 +23,24 @@ class MultipleSurveyError(Exception):
 
 
 @errors_blueprint.app_errorhandler(401)
-@errors_blueprint.app_errorhandler(CSRFError)
 @errors_blueprint.app_errorhandler(NoTokenException)
 def unauthorized(error=None):
     log_exception(error, 401)
     return render_template('session-expired.html'), 401
 
 
+@errors_blueprint.app_errorhandler(CSRFError)
+def csrf_error(error=None):
+    metadata = get_metadata(current_user)
+    if metadata:
+        if check_multiple_survey(metadata, request.view_args):
```

Comment:
Might be a little cleaner to combine these `if`s with an `and`

Reply:
I noticed that this morning and thought i'd been stupid not to do it at the time.

Reply:
Made the change but was naughty and squashed at the same time since it was pretty minor.